JuiceMaker/JuiceMaker/Controller/EditFruitsViewController.swift

```diff
@@ -1,13 +1,99 @@
+    @IBOutlet weak private var kiwiCountLabel: UILabel!
+    @IBOutlet weak private var mangoCountLabel: UILabel!
+        
+    @IBOutlet weak private var strawberryCountStepper: UIStepper!
+    @IBOutlet weak private var bananaCountStepper: UIStepper!
+    @IBOutlet weak private var pineappleCountStepper: UIStepper!
+    @IBOutlet weak private var kiwiCountStepper: UIStepper!
+    @IBOutlet weak private var mangoCountStepper: UIStepper!
+    
+    var fruits: Fruits?
     
     override func viewDidLoad() {
         super.viewDidLoad()
+        
+        setupFruitsCountSteppers()
+        setupFruitsCountLabels()
+    }
+    
+    private func setupFruitsCountSteppers() {
+        strawberryCountStepper.value = Double(fruits?[.strawberry] ?? 0)
+        bananaCountStepper.value = Double(fruits?[.banana] ?? 0)
+        pineappleCountStepper.value = Double(fruits?[.pineapple] ?? 0)
+        kiwiCountStepper.value = Double(fruits?[.kiwi] ?? 0)
+        mangoCountStepper.value = Double(fruits?[.mango] ?? 0)
+    }
+    
+    private func setupFruitsCountLabels() {
+        strawberryCountLabel.text = Int(strawberryCountStepper.value).description
+        bananaCountLabel.text = Int(bananaCountStepper.value).description
+        pineappleCountLabel.text = Int(pineappleCountStepper.value).description
+        kiwiCountLabel.text = Int(kiwiCountStepper.value).description
+        mangoCountLabel.text = Int(mangoCountStepper.value).description
+    }
+    
+    @IBAction private func pressStepper(_ sender: UIStepper) {
+        setNewValue(sender)
+        NotificationCenter.default.post(
+            name: NSNotification.Name.fruitsTag,
+            object: nil,
+            userInfo: [Constant.userInfoKey: fruits ?? [:]]
+        )
+    }
+    
+    private func setNewValue(_ stepper: UIStepper) {
+        switch stepper {
+        case strawberryCountStepper:
+            let strawberryValue = Int(strawberryCountStepper.value)
+            refreshFruitCountLabels(stepper, value: strawberryValue)
+            fruits?[.strawberry] = strawberryValue
+        case bananaCountStepper:
+            let bananaValue = Int(bananaCountStepper.value)
+            refreshFruitCountLabels(stepper, value: bananaValue)
+            fruits?[.banana] = bananaValue
+        case pineappleCountStepper:
+            let pineappleValue = Int(pineappleCountStepper.value)
+            refreshFruitCountLabels(stepper, value: pineappleValue)
+            fruits?[.pineapple] = pineappleValue
+        case kiwiCountStepper:
+            let kiwiValue = Int(kiwiCountStepper.value)
+            refreshFruitCountLabels(stepper, value: kiwiValue)
+            fruits?[.kiwi] = kiwiValue
+        case mangoCountStepper:
+            let mangoValue = Int(mangoCountStepper.value)
+            refreshFruitCountLabels(stepper, value: mangoValue)
+            fruits?[.mango] = mangoValue
+        default:
+            return
+        }
```

Comment:
모든 경우에 stepper의 value가 필요하다면 switch문의 스코프 바깥에서 처리할 수도 있겠네요. stepper와 과일의 종류를 매핑할 수 있는 방법을 만든다면 이렇게 작성할 수 있을 것 같아요.

```suggestion
       let fruitCount = Int(stepper.value)
       refreshFruitCountLabels(stepper, value: fruitCount)
       fruits?[stepper.asFruit] = fruitCount
```

Reply:
케이스를 분리해야 한다는 생각에 너무 매몰돼있었던 건지도 모르겠네요😅